app/models/location.rb

```diff
@@ -21,9 +21,9 @@ class Location < ApplicationRecord
   scope :search_by_postcode, ->(postcode) { where("REPLACE(postcode, ' ', '') ILIKE ?", "%#{postcode.delete(' ')}%") }
   scope :search_by_name, ->(name) { where("name ILIKE ?", "%#{name}%") }
   scope :search_by, ->(param) { search_by_name(param).or(search_by_postcode(param)) }
-  scope :started, -> { where("startdate <= ?", Time.zone.today).or(where(startdate: nil)) }
+  scope :started, -> { where("locations.startdate <= ?", Time.zone.today).or(where(startdate: nil)) }
   scope :active, -> { where(confirmed: true).and(started) }
-  scope :started_in_2_weeks, -> { where("startdate <= ?", Time.zone.today + 2.weeks).or(where(startdate: nil)) }
+  scope :started_in_2_weeks, -> { where("locations.startdate <= ?", Time.zone.today + 2.weeks).or(where(startdate: nil)) }
```

Comment:
Do we need to do this?

Reply:
Yep, when we join schemes and locations there's ambiguity now as they both have a startdate attribute, so this is needed to resolve that